### PR TITLE
Build bridge UI using Jenkins CI

### PR DESCRIPTION
**Pull request by gfournierPro**
The UI docker image is directly build with the help of Jenkins 

image

The version of the UI is the current running on the bridge

* test on local UI : 
https://goerli.etherscan.io/tx/0xb14d77c2f9cb734b3f4bebc51f8c0464544026db2fa1ab147c51cbe5560017dd

### CODE REVIEW

File: ui/.env.example [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
 REACT_APP_HOME_GAS_PRICE_SPEED_TYPE=standard
-REACT_APP_HOME_GAS_PRICE_FALLBACK=5000000000
+REACT_APP_HOME_GAS_PRICE_FALLBACK=0
 REACT_APP_HOME_GAS_PRICE_UPDATE_INTERVAL=15000
 REACT_APP_HOME_GAS_PRICE_FACTOR=1
 
-REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL=https://gasprice.poa.network/
-REACT_APP_FOREIGN_GAS_PRICE_SPEED_TYPE=standard
+REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL=https://eth-goerli.blockscout.com/api/v1/gas-price-oracle
+REACT_APP_FOREIGN_GAS_PRICE_SPEED_TYPE=average
```
Comment (james-toussaint) [inline ui/.env.example:27]: To understand: Issue with `standard`?

File: ui/.env.example [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
 REACT_APP_FOREIGN_GAS_PRICE_FALLBACK=5000000000
 REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL=15000
 REACT_APP_FOREIGN_GAS_PRICE_FACTOR=1
 
 # Default
-REACT_APP_TITLE=TokenBridge UI app - %c
-REACT_APP_DESCRIPTION=The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner.
+REACT_APP_TITLE=TokenBridge  UI app - %c
```
Comment (james-toussaint) [inline ui/.env.example:33]: Why space?

File: ui/.env.example [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+REACT_APP_DESCRIPTION=The Viviani bridge serves as a method of transferring Testnet RLC from the GOERLI to vRLC Native Viviani iExec sidechain in a quick and cost-efficient manner.
```
Comment (james-toussaint) [inline ui/.env.example]: Minor
```suggestion
REACT_APP_DESCRIPTION=The Viviani bridge serves as a method of transferring Testnet RLC from Goerli to the iExec Viviani sidechain (vRLC) in a quick and cost-efficient manner.
```

File: ui/Dockerfile [hunk 4]
```diff
@@ -1,21 +1,55 @@
-FROM node:8
+FROM node:10 as contracts
 
 WORKDIR /mono
-COPY package.json .
-COPY contracts/package.json ./contracts/
+
+COPY contracts/package.json contracts/package-lock.json ./contracts/
```
Comment (james-toussaint) [inline ui/Dockerfile]:
```suggestion
WORKDIR /mono/contracts
COPY contracts/package*.json ./
RUN npm install --only=prod
```
?

File: ui/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+buildInfo = getBuildInfo()
+
+buildSimpleDocker_v3(
+    buildInfo: buildInfo,
+    visibility: Registries.EXTERNAL_IEXEC_HOST,
+    dockerfileDir: './ui',
+    buildContext: '.',
+    dockerImageRepositoryName : 'bridge-ui',
+)
```
Comment (james-toussaint) [inline ui/Jenkinsfile]: Minor
```suggestion
)

```

File: ui/Dockerfile [hunk 6]
```diff
@@ -1,21 +1,55 @@
...
-RUN yarn run compile:contracts
-RUN mv ./contracts/build ./ && rm -rf ./contracts/* ./contracts/.[!.]* && mv ./build ./contracts/
+WORKDIR /mono/oracle
+RUN yarn remove web3 web3-utils
+
+WORKDIR /mono
+COPY --from=contracts /mono/contracts/build ./contracts/build
+COPY ui/package.json ui/lib/web3-eth/index.js ./ui/
+
+WORKDIR /mono
```
Comment (james-toussaint) [inline ui/Dockerfile]:
```suggestion
```

File: ui/Dockerfile [hunk 7]
```diff
@@ -1,21 +1,55 @@
...
+
+RUN rm -rf yarn.lock 
+COPY yarn.lock ./
```
Comment (james-toussaint) [inline ui/Dockerfile]: Why do we need that?

File: ui/.env.example (hunk 1 above)
Comment (gfournierPro) [reply ui/.env.example:27]: yes it does not exist on the query `{"average":0.3,"fast":1.58,"slow":0.2}`

**Review by james-toussaint — APPROVED** [review]
Ok thanks :slightly_smiling_face:

**Pull request merged**